test/python/test_parallel.py

```diff
@@ -24,6 +24,11 @@ def _parfunc(x):
 class TestParallel(QiskitTestCase):
     """A class for testing parallel_map functionality.
     """
+
+    def test_parallel_env_flag(self):
+        """Verify parallel env flag is set """
+        self.assertEqual(os.getenv('QISKIT_IN_PARALLEL', None), 'FALSE')
```

Comment:
Probably should just use self.assertFalse() here instead.

Reply:
It is not a bool `False` but rather the string `'FALSE'`.

Reply:
Oh nevermind, ignore me, I can't read today. Completely missed the quotes and that this is a string comparison.